src/icons/raw/Heart.svg

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="18px" height="15px" viewBox="0 0 18 15" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <g id="Symbols" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="Icons-/-Heart-/-Empty" transform="translate(-3.000000, -5.000000)" stroke="#000000">
```

Comment:
I think it's best to remove the `id`'s.
They might be removed automatically by Yoshi, but just to be super sure, let's remove them.

Reply:
SVGs was minified.